Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml

```diff
@@ -30,30 +33,44 @@
                 </StackPanel>
 
                 <Grid Margin="5,10,5,0" >
+                    <Grid.Resources>
+                        <local:BooleanToVisiblityCollapsedConverter x:Key="visibilityConverter" />
+                    </Grid.Resources>
                     <Grid.ColumnDefinitions>
                         <ColumnDefinition Width="50*"></ColumnDefinition>
                         <ColumnDefinition Width="50*"></ColumnDefinition>
                     </Grid.ColumnDefinitions>
 
-                    <StackPanel Grid.Column="0" HorizontalAlignment="Left" Orientation="Horizontal">
+                    <StackPanel Grid.Column="0" Grid.Row="0" Margin="5,0,0,0" HorizontalAlignment="Left" Orientation="Horizontal">
                         <Label Style="{StaticResource $LabelStyle}" Content="Target Application:"/>
-                        <UserControlsLib1:UCComboBox x:Name="xTargetApplicationComboBox" VerticalAlignment="Top" Width="250"></UserControlsLib1:UCComboBox>
+                        <ComboBox x:Name="xTargetApplicationComboBox" Style="{StaticResource $FlatInputComboBoxStyle}" SelectionChanged="xTargetApplicationComboBox_SelectionChanged" VerticalAlignment="Top" Width="250" ></ComboBox>
                     </StackPanel>
 
-                    <StackPanel Grid.Column="0" HorizontalAlignment="Left" Orientation="Horizontal">
-                        <Label Content="Page URL:" Style="{StaticResource $LabelStyle}" HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                        <TextBox x:Name="xPageURLTextBox" Style="{StaticResource $TextBoxStyle}" HorizontalAlignment="Left" VerticalAlignment="Center" FontWeight="Bold" TextWrapping="NoWrap" Width="250" Margin="45,0,0,0"/>
-                        <UserControls:ucButton x:Name="xPageURLBtn" ButtonType="ImageButton" HorizontalAlignment="Center" ButtonFontImageSize="16" ButtonImageWidth="16" ButtonImageHeight="16" ButtonImageType="GoNext" Click="xPageURLBtn_Click" ToolTip="Go to Page"/>
+                    <agents:ucAgentControl x:Name="xAgentControlUC" Grid.Column="1" Grid.Row="0" AgentStartedUCEvent="AgentStartedHandler" HorizontalAlignment="Right" Width="350" VerticalAlignment="Top" Margin="0,-5,0,0" ></agents:ucAgentControl>
+
+                    <StackPanel Grid.Column="0" Margin="0,30,0,0" HorizontalAlignment="Left" >
+                        <Label Margin="5,0,0,0" Style="{StaticResource $LabelStyle}" Content="Page Load Flow:" FontWeight="Bold" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                        <RadioButton x:Name="xRDBPageURL" Height="20" Content="Page URL"  Grid.Column="0" Grid.Row="1" Tag="radPageURL" Checked="XRDBPageURL_Checked" HorizontalAlignment="Left" Margin="10,5,0,0" />
```

Comment:
same names and events comments like on POMGeneralDetails xaml

Reply:
changed